src/RadixDlt.NetworkGateway.PostgresIntegration/LedgerExtension/PostgresLedgerExtenderService.cs

```diff
@@ -273,12 +278,13 @@ private async Task<ExtendLedgerReport> ProcessTransactions(ReadWriteDbContext db
 
                     if (committedTransaction.LedgerTransaction is CoreModel.UserLedgerTransaction userLedgerTransaction)
                     {
-                        ledgerTransactionMarkersToAdd.Add(new OriginLedgerTransactionMarker
-                        {
-                            Id = sequences.LedgerTransactionMarkerSequence++,
-                            StateVersion = stateVersion,
-                            OriginType = LedgerTransactionMarkerOriginType.User,
-                        });
+                        ledgerTransactionMarkersToAdd.Add(
```

Comment:
Looks like some automatic code-reformatter kicked in? There's no change or anything here and in other places just the formatting?

Reply:
there are some code changes here, if you set ignore whitespaces it looks way better, although you'll still see some reformatting.